modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldValueRenderer.java

```diff
@@ -47,10 +47,11 @@ public String render(DDMFormFieldValue ddmFormFieldValue, Locale locale) {
 		}
 
 		try {
-			FileEntry fileEntry = dlAppService.getFileEntryByUuidAndGroupId(
-				uuid, groupId);
+			DLFileEntry dlFileEntry =
+				dlFileEntryLocalService.getFileEntryByUuidAndGroupId(
+					uuid, groupId);
```

Comment:
Why did you not use the `DLAppLocalServiceImpl` class?

Reply:
Hi @marcelabc, by using `DLAppLocalServiceImpl` we get the following:

>{&#34;extension&#34;:&#34;jpg&#34;,&#34;size&#34;:&#34;426615&#34;,&#34;groupId&#34;:&#34;20117&#34;,&#34;classNameId&#34;:33190,&#34;title&#34;:&#34;39610.jpg&#34;,&#34;type&#34;:&#34;document&#34;,&#34;uuid&#34;:&#34;99d16bcb-b18e-8c5a-656a-770a3bfef4a6&#34;,&#34;fileEntryId&#34;:&#34;33611&#34;,&#34;url&#34;:&#34;/documents/portlet_file_entry/20117/39610.jpg/99d16bcb-b18e-8c5a-656a-770a3bfef4a6?imagePreview=1&#34;}

When we should actually just get the file name, therefore, I used `DLAppLocalService` because we're able to do so.

Reply:
Hi @marcelabc! What are your thoughts about this?